src/server.ts

```diff
@@ -11,7 +12,11 @@ const app = express();
 app.use(express.json());
 
 createConnection().then(() => {
-  app.use("/api", StudentRouter().getAssembledRouter());
+  app.use(
+    "/api",
+    StudentRouter().getAssembledRouter(),
+    UserRouter().getAssembledRouter()
+  );
```

Comment:
Veo que Eyder colocó su router en un app.use separado, así que lo podemos manejar así:

```suggestion
  app.use("/api", StudentRouter().getAssembledRouter());
  app.use("/api", UserRouter().getAssembledRouter());
```